lib/cosmos/tools/tlm_viewer/widgets/widget.rb

```diff
@@ -258,5 +258,20 @@ def context_menu
       point.dispose
       menu.dispose
     end
+
+    # Requires the @screen to be set so must not be called in initialize()
+    def get_image(image_name)
+      return nil unless @screen
+      target_screen_dir = File.join(::Cosmos::USERPATH, 'config', 'targets', @screen.original_target_name.upcase, 'screens')
+      cosmos_data_dir = File.join(::Cosmos::USERPATH, 'config', 'data')
+
+      if File.exist?(File.join(target_screen_dir, image_name))
+        return Qt::Image.new(File.join(target_screen_dir, image_name))
```

Comment:
I think putting the images into the target screen folder makes more sense and better supports our philosophy of having a self contained target folder.

Reply:
Feels like this should use top_level#data_path